views/layout.pug

```diff
@@ -2,24 +2,25 @@ doctype html
 html(lang="en")
 	head
 		meta(charset="utf-8")
+		base(href="/btc-rpc-explorer/")
```

Comment:
I believe you intended to use variable here. It breaks when e.g. `/btc-explorer/` is used

Reply:
good find!